pkg/virt-launcher/virtwrap/agent-poller/agent_parser.go

```diff
@@ -168,20 +70,19 @@ func ParseFSFreezeStatus(agentReply string) (api.FSFreeze, error) {
 	}, nil
 }
 
-// parseTimezone from the agent response
-func parseTimezone(agentReply string) (api.Timezone, error) {
-	result := Timezone{}
+// parseAgent gets the agent version from response
+func parseAgent(agentReply string) (AgentInfo, error) {
```

Comment:
nit: this function was moved to a different line in this file. Can you keep it in the same place? It would be easier to see that nothing has changed.

Reply:
I'm not sure how to do this. This file was reduced from 298 lines to 125 lines.

Reply:
I mean, in the diff view currently it looks like this function `parseAgent()` was created by changing function `parseTimezone()`. If you would move this `parsAgent()` back to the place where it was before the change, the diff would only show removal of lines.
Simply move it at the end of the file.

Reply:
Ah I see. Thanks.